lib/oxidized/node.rb

```diff
@@ -155,11 +155,13 @@ def resolve_input opt
     end
 
     def resolve_output opt
-      output = (opt[:output] or Oxidized.config.output.default)
-      if not Oxidized.mgr.output[output]
-        Oxidized.mgr.add_output output or raise MethodNotFound, "#{output} not found for node #{ip}"
+      outputs = (opt[:output] or Oxidized.config.output.default)
```

Comment:
Re: consistent naming: well, this part is really copy/paste from 'resolve_input'. So if I change this it will not be consistent with the rest of the code. And I wanted to keep the changes to a minimum.